modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/internal/model/listener/test/UserModelListenerTest.java

```diff
@@ -139,11 +183,17 @@ private void _assertEqualsSorted(
 	private static final String _OBJECT_DEFINITION_NAME =
 		ObjectDefinitionTestUtil.getRandomName();
 
+	private AuditMessageProcessor _auditMessageProcessor;
+	private ServiceRegistration<AuditMessageProcessor>
+		_auditMessageProcessorServiceRegistration;
+	private BundleContext _bundleContext;
 	private Company _company;
 
 	@Inject
 	private CompanyLocalService _companyLocalService;
 
+	private User _defaultUser;
```

Comment:
The default user is only used inside the method `testOnAfterRemove` I believe it's more appropriate create him inside the method scope.